### PR TITLE
Cm trt1

### PR DESCRIPTION
**Pull request by eyalpe**

### CODE REVIEW

File: include/cppflowCM/InterfaceCM.h [hunk 1]
```diff
@@ -1,47 +1,38 @@
...
-  int m_patchHeight = 128;
-  int m_patchWidth = 128;
+  // int m_patchHeight = 128;
+  // int m_patchWidth = 128;
+  // int m_batchSize = 32;
+  // int m_numColors = 7;
+  // bool m_hardBatchSize = false;
+  //float m_tileMargin = 0.2;
 
-#ifdef TEST_MODE
```
Comment (borisef) [inline include/cppflowCM/InterfaceCM.h:20]: I need this test mode

File: include/cppflowCM/InterfaceCMbase.h [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                                                 OD::e_OD_TargetColor::WHITE,   // 5
+                                                 OD::e_OD_TargetColor::YELLOW}; // 6
+    static int num_colors = sizeof(cid_to_enum) / sizeof(OD::e_OD_TargetColor); //num of colors in the array
+    if (cid < num_colors)
+    {
+      ans = cid_to_enum[cid];
+    }
+
+#ifdef TEST_MODE
+    // map color id to color-name
```
Comment (eyalpe) [inline include/cppflowCM/InterfaceCMbase.h:110]: take color name from odutils

File: CMakeLists.txt [hunk 3]
```diff
@@ -11,7 +11,7 @@ message($ENV{USER})
 
 
 
-set(TEST_MODE TRUE)
+#set(TEST_MODE TRUE)
```
Comment (borisef) [inline CMakeLists.txt]: make test_mode FALSE instead

File: CMakeLists.txt [hunk 4]
```diff
@@ -63,8 +63,9 @@ if(OS_WINDOWS)
     set(MY_TF_LIB2 "")
 else(OS_WINDOWS) #Linux
     include_directories(PRIVATE include)
-    find_package( OpenCV REQUIRED)
-    
+    set(OpenCV_DIR /usr/lib/x86_64-linux-gnu/cmake/opencv4)
```
Comment (borisef) [inline CMakeLists.txt]: avoid x86_64

**Review by borisef — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 3 above)
Comment (eyalpe) [reply CMakeLists.txt]: done

File: include/cppflowCM/InterfaceCM.h (hunk 1 above)
Comment (eyalpe) [reply include/cppflowCM/InterfaceCM.h:20]: resolved, but in 'src/Object_Detection_Handler.cpp'.

**Pull request merged**